integrations/integration-vip-config.php

```diff
@@ -135,12 +135,14 @@ public function get_site_status() {
 	 * Get site config.
 	 *
 	 * @return array
-	 *
-	 * @private
 	 */
 	public function get_site_config() {
 		if ( is_multisite() ) {
 			$config = $this->get_value_from_config( 'network_sites', 'config' );
+			// If network site config is not found then fallback to env config if it exists
```

Comment:
maybe not applicable to this line. But in the GOOP PR, I see that when both the network site and main site config exist, we merge them:

https://github.com/Automattic/vip-go-api/pull/5172/files#diff-8e75ccc1dc1ca94f37c3361246e36acd19c64eee88d78eafe15ed8b040734ce1R392

Do we need to do anything to keep that behavior in mu-plugins, or does that get encoded in the configmap for that network site??

Reply:
@noahtallen That gets encoded in the ConfigMap for the network site!